identity-chaining-draft.md

```diff
@@ -254,9 +254,9 @@ The client MAY indicate the audience it is trying to access through the `scope`
 
 ### Processing rules
 
-All of {{RFC7521}} (Section 5.2 in specific) applies, along with the following processing rules:
+All of Section 5.2 {{RFC7521}} applies, in addition to the following processing rules:
 
-* The request MUST be denied if the presented authorization grant does not include an "aud" claim identifying the authorization server that processes the request.
+* The "aud" claim value MUST be either the token endpoint identifier as described Section 3 {{RFC7523}} or the Authorization Server issuer idetifier as defined in Section 5.2 of {{RFC7521}}. 
```

Comment:
idetifier -> identifier  

but Section 5.2 of RFC7521 doesn't define the issuer identifier, does it? I think "issuer identifier" is defined first in AS Metadata RFC8414

Reply:
https://datatracker.ietf.org/doc/html/rfc9126#section-2-5 or https://www.rfc-editor.org/rfc/rfc9207#section-1.1-2 are examples of places that cite it

Reply:
Aha! That makes a lot more sense, thanks Brian. Let me try again.

Reply:
@bc-pi I made updates to try and clarify that:

1. The aud claim should identify the token endpoint or the authorization server
2. The issuer identifier should follow the format defined in RFC 8414

Thanks for pointing that out, I would never have known to look there otherwise.

Please suggest alternative text in case the change is not clear.

Reply:
Thinking about it a bit more and this is trickier than it should be- sorry! But I think the language you had before is preferable with just getting the reference and spelling right. I'll suggest some text

Reply:
I'm even kinda having second thoughts about the value/appropriateness of bringing up RFC 8414 here but it's probably okay...